### DISCUSSION TITLE
Expired token

### DISCUSSION THREAD

**Discussion by mothana-alabsi** [elsa-workflows/elsa-core#4052] [General]
When I create the procedure flow, I need to create two options, so I used the fork to create two signals, but when using the first signal and returning to use the second signal after six activities, it shows the expiration time for the other signal. Is there a solution? or anyone face this problem.

**Comment by sfmskywalker** [comment]
That doesn't ring any bells. Perhaps you can share a workflow JSON file that I can import and try for myself?
